src/library_async.js

```diff
@@ -258,14 +276,14 @@ mergeInto(LibraryManager.library, {
       dbg('ASYNCIFY: setDataRewindFunc('+ptr+'), bottomOfCallStack is', bottomOfCallStack, new Error().stack);
 #endif
       var rewindId = Asyncify.getCallStackId(bottomOfCallStack);
-      {{{ makeSetValue('ptr', C_STRUCTS.asyncify_data_s.rewind_id, 'rewindId', 'i32') }}};
+      {{{ makeSetValue('ptr', C_STRUCTS.asyncify_data_s.rewind_id, 'rewindId', '*') }}};
```

Comment:
`rewind_id` is defined as an `int` in `emscripten/fiber.h`.. so this seems wrong, no?